pipelinewise/fastsync/commons/tap_mongodb.py

```diff
@@ -192,13 +232,17 @@ def copy_table(self,
 
                 # bson.decode_file_iter will generate one document at a time from the exported file
                 for document in bson.decode_file_iter(export_file):
-                    rows.append({
-                        '_ID': str(document['_id']),
-                        'DOCUMENT': json.dumps(document, cls=MongoDBJsonEncoder, separators=(',', ':')),
-                        utils.SDC_EXTRACTED_AT: extracted_at,
-                        utils.SDC_BATCHED_AT: datetime.datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S.%f'),
-                        utils.SDC_DELETED_AT: None
-                    })
+                    try:
+                        rows.append({
+                            '_ID': str(document['_id']),
+                            'DOCUMENT': ujson.dumps(serialize_document(document)),
```

Comment:
why the new ujson? Is that for the extra performance reason or it's serialising some items differently?

Reply:
it's the former, ujson is faster than the built in json module.

Reply:
looks like a small effort task that we can do with all taps/targets. Do you know anything why we should not switch to ujson in every component? Obviously not part of this PR, just wondering.

Reply:
I wouldn't call it small effort task, it's indeed a small change but with the number of taps/target we have it's painful hahaha